apps/address-book/arapp.json

```diff
@@ -23,6 +23,12 @@
       "wsRPC": "wss://rinkeby.eth.aragon.network/ws",
       "network": "rinkeby"
     },
+    "continuous-deployment": {
```

Comment:
Given how similar this is to the `rinkeby` setup above, can you explain more about why we need both?

Reply:
> Given how similar this is to the `rinkeby` setup above, can you explain more about why we need both?

Note the app name is different for this environment, I guess that makes the need to separate it from rinkeby, I find that a good idea atm

Reply:
The concept of an `environment` exists separately from a `network` for this very purpose. You can have multiple namespaces (ie `appName`s) of a single app on a given network. For example, we have our public `rinkeby` deploy which is what is available in the App center, and we also have `staging`, which exists as our public demo-dao namespace. The `continuous-deployment` namesapace serves its own distinct purpose of giving us a distinct publishing space that doiesn't touch our public-facing deployments, while also existing on rinkeby and utilizing the same registry

Reply:
I was picturing this as our eventual goal:

* merges to `dev` automatically deploy to our `staging` environment, and anyone using our Rinkeby apps can get those changes immediately
* merges to `master` automatically deploy to our `production` environment; anyone using our mainnet apps can get those

I'm not certain we need the flexibility+overhead (they go together, in this case) of having two separate Rinkeby namespaces. Do you see having both as temporary?

Reply:
I think our automated testing apparatus would have to be much more thorough to reach this point